tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,33 @@
+const GOLF_SCORES = {
```

Comment:
Just for information. You can export like this as well, without exporting object at the end
```export const GOLF_SCORES = {```

```export function golfScore(par, strokes) {```